v5/api/c/rotation.rst

```diff
@@ -149,7 +149,9 @@ Analogous to `pros::Rotation::reset_position <../cpp/rotation.html#reset-positio
 rotation_get_position
 ---------------------
 
-Get the Rotation Sensor's current position in centidegrees
+Get the Rotation Sensor's current angle in centidegrees (0-36000).  This means the absolute position
```

Comment:
As someone who has not worked with this sensor before, the following strikes me as a bit of a clearer wording of the same concept:
```
Get the Rotation Sensor's angle _relative to its starting position_ in centidegrees (1/100th of a degree, or 1/36000th of a rotation). This angle is measured in relation to its 0 centidegree position when the sensor was last initialized or reset. This reading will _not_ roll over when crossing a full rotation, but will continue to increment or decrement.
```

Reply:
I agree. The current wording implies that the value will roll over, when this is not actually the case.